### ISSUE TITLE
False positive Bundler/DuplicatedGem

### ISSUE THREAD

**Issue by marcandre** [rubocop-hq/rubocop#8604]
```ruby
local = File.expand_path('../rubocop', __dir__)
if Dir.exist? local
  gem 'rubocop', path: local 
else
  gem 'rubocop', '~> 0.90.0'
end
```

Should not register an offense.
